e2e/suites/extensions/ext-document-list.test.ts

```diff
@@ -84,7 +84,7 @@ describe('Extensions - DocumentList presets', () => {
     const expectedColumns = testData.filter((item) => !item.disabled).map((data) => data.label);
     const actualColumns = await dataTable.getColumnHeadersText();
 
-    expect(actualColumns).toEqual(expectedColumns);
+    await expect(actualColumns).toEqual(expectedColumns);
```

Comment:
why do you add await here, and only in some places?
I'd say you either add it for every single expect from the project, or not at all.

Reply:
it's like saying "either test the whole app or don't test it at all" :) I would prefer the iteration approach and continuous improvements :) contributing just the pieces that making sad one of the code analysis tool I'm testing, this is not the ticket/task to change all

Reply:
true :)

maybe we can see if we can remove @types/jasminewd2 all together, and leave only jasmine types, that would solve the need for await in front of expect